code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -80,7 +80,6 @@
 	new /obj/item/weapon/gun/energy/gun/martin(src)
 	new /obj/item/clothing/accessory/holster/waist(src)
 	new /obj/item/weapon/melee/telebaton(src)
-	new /obj/item/clothing/head/beret/sec/navy/hos(src)
```

Comment:
i only see one in this list and you've removed it, where is this second beret?

Reply:
![cb6f54c5-fdc8-4114-8901-5fbf1fe9debf](https://user-images.githubusercontent.com/22408776/52500764-5f005900-2ba4-11e9-9a52-937c83245643.jpeg)